src/application/router/routes.ts

```diff
@@ -181,19 +181,20 @@ const routes: RouteRecordRaw[] = [
   },
 
   /**
-   * 404 page
+   * error page
    */
   {
-    path: '/:pathMatch(.*)*',
+    path: '/error',
     component: ErrorPage,
     meta: {
       layout: 'fullpage',
-      pageTitleI18n: 'pages.notFound',
+      pageTitleI18n: 'pages.error',
       discardTabOnLeave: true,
     },
-    props: {
-      code: 404,
-    },
+    props: route => ({
+      code: route.query.code,
+      customMessage: route.query.message,
+    }),
```

Comment:
404 page was actually needed (this was a syntax for each unexpected url, that would show 404 `page not found`)

we need to create separate /error page, that should have `statusCode` property and show different info by different status code

Reply:
also better to use route.params for `code` prop